web/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Kalkulator miar</title>
+    <style type="text/css">
+        form {
+            width: 95%;
+            margin-left: auto;
+            margin-right: auto;
+        }
+    </style>
+</head>
+<body>
+<form method="get" action="calcmetryczny">
+    <h1>Przelicznik metryczny</h1>
+    <input type="text" name="metry" placeholder="metry"><br/>
+    <input type="text" name="centymetry" placeholder="centymetry"><br/>
+    <input type="text" name="milimetry" placeholder="milimetry"><br/>
+    <button type="submit">Przelicz</button>
+</form>
+<form method="get" action="calcwag">
```

Comment:
Tutaj mógłbyś też użyć `POST`, ale tutaj to kwestia tego czy będziemy się tym dzielić z kimś innym czy nie :)